source/euclid.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const gcd = (x, y) => {
+	while (x > 0 & y > 0) {
+		if (x > y) {
+			x = x % y;
+		}
+		else {
+			y = y % x;
+		}
+	}
+	
+	return x + y;
+}
+
+const euclid = (...numbers) => {
+	if (numbers.some((number) => !Number.isFinite(number) ? true : false)) {
```

Comment:
```suggestion
	if (numbers.some((number) => !Number.isFinite(number))) {
```
бесполезное `... ? true : false` - расскажи почему

Reply:
а ещё я тут подумал - могут же быть ещё дробные числа - поменяй на `Number.isInteger()` + тест поправь на проверку этого + не забудь о `NaN`

Reply:
первая половина. у нас же isFinite итак возвращает bool, который мы потом инвертируем

Reply:
я же правильно понял, что встроенного isInteger() нет? на learn javascript нет.

а NaN уже включен в проверку isFinite

Reply:
learnjs это всё таки больше про научить, а для найти есть mdn https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger